modules/configuring-machine-pool-disk-volume-ocm.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+ifdef::openshift-rosa[]
+[id="configuring_machine_pool_disk_volume_ocm{context}"]
+= Configuring machine pool disk volume using OpenShift Cluster Manager
+endif::openshift-rosa[]
+.Prerequisite for cluster creation
+* You have the option to select the node disk sizing for the default machine pool during cluster installation.
+
+.Procedure for cluster creation
+
+. From ROSA cluster wizard, navigate to Cluster settings.
+
+. Navigate to *Machine pool tab*.
```

Comment:
No machine pool tab in wizard hence please correct the sentence as below.
Navigate to Machine pool step.